pandas/core/window.py

```diff
@@ -963,8 +963,19 @@ def count(self):
         not passed. In the future `raw` will default to False.
 
         .. versionadded:: 0.23.0
+    *args, **kwargs :
+        Arguments and keyword arguments to be passed into func.
 
-    \*args and \*\*kwargs are passed to the function""")
+    Returns
```

Comment:
Actually looking at this in more detail - why do this here instead of updating `_doc_template` with this?

Reply:
Hey @WillAyd, 
I did it in the ```_shared_docs['apply']``` because the ```@Appender(_doc_template)``` decorator on the window.Expanding/Rolling.apply methods is what _threw_ the original validation error ```No Returns section found```. I was also not sure whether my changes, for window.Expanding/Rolling.apply, are relevant for the other functions wrapped with ```@Appender(_doc_template)```. 

I see your point though, I am happy to update the ```_doc_template``` 
From:
```
_doc_template = """
        Returns
        -------
        same type as input

        See Also
        --------
        Series.%(name)s
        DataFrame.%(name)s
"""
```
To:
```
_doc_template = """
        Returns
        -------
        Series or DataFrame
                Return type is determined by the caller.

        See Also
        --------
        Series.%(name)s : Series %(name)s.
        DataFrame.%(name)s : DataFrame %(name)s.
"""
```

. . . if you agree with my modification to ```_doc_template ```?

Reply:
You should be able to remove this all now that you've made these changes to `_doc_template`